Makefile

```diff
@@ -0,0 +1,14 @@
+.PHONY: help pre-commit clean clean-build clean-pyc clean-test venv
```

Comment:
this line is also in the py makefile, does it need to have the same commands in both places?

Reply:
i believe so. if someone were to do `make -f pytest-selenium/Makefile-py`

Reply:
However, that is not likely so i'll remove it ;)